app/src/main/java/com/nervousfish/nervousfish/activities/WaitActivity.java

```diff
@@ -33,6 +34,7 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger("WaitActivity");
     private IServiceLocator serviceLocator;
+    private IDatabase database;
```

Comment:
Not sure if you need this attribute since you're only using it in the `onNewDataReceivedEvent()` method 🤔